packages/kit/src/views/Staking/pages/ProtocolDetailsV2/index.tsx

```
@@ -92,328 +137,389 @@ function SubscriptionSection({
-            justifyContent="space-between"
-          >
-            <XStack alignItems="center" gap="$1.5">
-              <Token size="sm" tokenImageUri={item.token.logoURI} />
-              <NumberSizeableText
-                size="$bodyLgMedium"
-                formatter="balance"
-                formatterOptions={{ tokenSymbol: item.token.symbol }}
+      {rewards?.tokens?.map((token, index) => {
+        return (
+          <>
+            <YStack gap="$2.5">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)